src/main/kotlin/com/r3/developers/csdetemplate/MyFirstFlow.kt

```diff
@@ -14,8 +14,7 @@ import net.corda.v5.base.util.contextLogger
 class MyFirstFlowStartArgs(val otherMember: MemberX500Name)
 
 
-// A class which will contain a message, It must be marked with @CordaSerializable for Corda
-// to be able to send from one virtual node to another.
+// A class contains a message, mark it with @CordaSerializable for Corda to send one virtual node to another.
```

Comment:
From: "A class contains a message, mark it with @CordaSerializable for Corda to send one virtual node to another."

To: "Where a class contains a message, mark it with @CordaSerializable to enable Corda to send it from one virtual node to another."